src/Features/VisualBasic/Portable/CodeRefactorings/InvertIf/InvertIfCodeRefactoringProvider.MultiLine.vb

```diff
@@ -0,0 +1,59 @@
+﻿' Copyright (c) Microsoft.  All Rights Reserved.  Licensed under the Apache License, Version 2.0.  See License.txt in the project root for license information.
+
+Imports System.Composition
+Imports Microsoft.CodeAnalysis.CodeRefactorings
+Imports Microsoft.CodeAnalysis.Text
+Imports Microsoft.CodeAnalysis.VisualBasic.Syntax
+
+Namespace Microsoft.CodeAnalysis.VisualBasic.CodeRefactorings.InvertIf
+    <ExportCodeRefactoringProvider(LanguageNames.VisualBasic, Name:=PredefinedCodeRefactoringProviderNames.InvertIf), [Shared]>
+    Friend NotInheritable Class VisualBasicInvertMultiLineIfCodeRefactoringProvider
+        Inherits VisualBasicInvertIfCodeRefactoringProvider(Of MultiLineIfBlockSyntax)
+
+        Protected Overrides Function GetHeaderSpan(ifNode As MultiLineIfBlockSyntax) As TextSpan
+            Return TextSpan.FromBounds(
+                    ifNode.IfStatement.IfKeyword.SpanStart,
+                    ifNode.IfStatement.Condition.Span.End)
+        End Function
+
+        Protected Overrides Function IsElseless(ifNode As MultiLineIfBlockSyntax) As Boolean
+            Return ifNode.ElseBlock Is Nothing
+        End Function
+
+        Protected Overrides Function CanInvert(ifNode As MultiLineIfBlockSyntax) As Boolean
+            Return ifNode.ElseIfBlocks.IsEmpty
```

Comment:
This should be removed since it is causing us to no longer offer invert if on single statement if-then-else statements.

Reply:
> on single statement if-then-else

If you mean "single line if statements", that's handled in a separate derived provider. we do offer invert-if for those.

Reply:
My mistake, this line of code is fine :)

I noticed some tests that used to offer invert if are no longer doing so, but the cause is elsewhere so I'll comment further there.